### ISSUE TITLE
Increased sim limit in Combat Simulator

### ISSUE THREAD

**Issue by 2023PerfectDark** [fgsfdsfgs/perfect_dark#147]
is there a way to add a sim slider in CS? This guy made one but I'm not sure if it's his own PC port edit or a decompilation edit

https://www.youtube.com/watch?v=u7UfGUVzSF0
![Screenshot 2023-09-04 at 00-20-11 (1) Perfect Dark N64 - Max Bots #4 - VS 100 Simulants - Carrington Villa (Decompilation Mod) - YouTube](https://github.com/fgsfdsfgs/perfect_dark/assets/142272040/ad5da98c-08e7-49af-868c-7f14ddae1332)

**Comment by fgsfdsfgs** [comment]
This is one of the [mod branches](https://github.com/n64decomp/perfect_dark/tree/mods/maxbots) of the decompilation project.  It is possible to include the relevant changes from that branch.
Coincidentally, it also includes a feature to let you spawn with a weapon (#144).

**Comment by Atari-Dude** [comment]
Has anyone been able to bring this increased sim limit slider mod over to the PC port yet?

**Comment by fgsfdsfgs** [comment]
I've looked into the maxbots branch a bit and it changes quite a lot. It's probably possible to increase the sim count to like 16 or something without too much of a pain though.

**Comment by 2023PerfectDark** [comment]
yeah, that sounds fine. try 16 and then increase it when you have time or until you run into problems or the game can't handle it.

**Comment by regionalradio** [comment]
Is 16 sims a possibility
